packages/ooth-jwt/src/index.ts

```diff
@@ -59,22 +42,50 @@ export default function({
   sharedSecret,
   privateKey,
   publicKey,
-  algorithm = 'RS256'
+  algorithm = 'RS256',
+  includeProfile = false,
 }: Options): void {
-  if(sharedSecret === undefined && privateKey === undefined) {
+  async function getToken(userId: string, iat: number, tokenExpiry: number, options: TokenOptions): Promise<string> {
```

Comment:
Thank you for your pr. Small change: could you keep this function outside as it was at the beginning?

Reply:
That would require ooth to be passed in as a parameter. Is that okay?

Reply:
Yes, no problem